src/packages/ratmama/ratmama_parser.py

```diff
@@ -152,8 +154,8 @@
 
     for rescue in ctx.bot.board.rescues.values():
         if rescue.irc_nickname.casefold() == ctx.user.nickname.casefold():
-            await ctx.reply(
-                "You already sent a signal, please be patient while a dispatch is underway.")
+            await ctx.reply(f"{ctx.user.nickname}: You already sent a Signal! Please stand by,"
```

Comment:
[error] *squint* doesn't `ctx.reply` already embed the nickname into the message stream?